### ISSUE TITLE
stand alone version (umd / cdn)

### ISSUE THREAD

**Issue by sag1v** [airbnb/react-dates#1471]
There were couple of issues (#375 #426) opened regarding exposing a UMD or a cdn version for `react-dates`.  
@ljharb referenced this link https://wzrd.in/standalone/react-dates@latest but it seems that its timed out and not accessible anymore. 

Is there any other source for this or plans to expose a build version as a cdn?

**Comment by ljharb** [comment]
While it’s unfortunate that wzrd.in isn’t reliable atm, that doesn’t change that amd and globals are obsolete, and that current best practices are to bundle up libs yourself.

We could add a “dist” build process but i don’t think it would add much value - it’d just be enabling people to stay on antiquated build pipelines for longer.

**Comment by sag1v** [comment]
Actually my use case was using it in stack snippets when writing answers with running code examples on stackoverflow

**Comment by ljharb** [comment]
That’s a good reason :-) i think #375 covers that use case?

**Comment by sag1v** [comment]
That's the same issue and outdated link i posted above though

**Comment by ljharb** [comment]
Yes, but since it remains open, i think this should probably be closed in favor of that one

**Comment by ljharb** [comment]
Ah, it’s closed. In that case, yes, a working codepen or jsfiddle would be useful.

**Comment by majapw** [comment]
Do any of the codepen examples in https://github.com/airbnb/react-dates/issues/426 still work? I guess it would be nice to have an official, react-dates codepen from which to work off of. :)

**Comment by sag1v** [comment]
@majapw None is working as  https://wzrd.in/standalone/react-dates@latest  is outdated.
I think the options are either host the a umd / cdn version (paid or free) in some web hosting or just expose a bundled file for unpkg or similar websites.

**Comment by aminpaks** [comment]
>amd and globals are obsolete, and that current best practices are to bundle up

Would you provide a reference?

**Comment by ljharb** [comment]
@aminpaks if you won't take my comment as sufficient reference, then I'm not sure why a link to some other internet rando's comment would help.

Relying on globals has been seen as pretty universally bad in the industry for well over a decade; as has "not using modules" for at least half a decade. Since a bundler is required to use modules (except with native ESM in the most modern browsers as of the last year or so, although native ESM is still too slow to be practical), a bundler is thus also a best practice.

**Comment by ljharb** [comment]
In my experience, using a third party cdn doesn’t actually add value if you’re small. If you’re large enough that it might, then it’d be more efficient to serve it yourself, on your own cdn (with your own bundle splitting technique).

**Comment by aminpaks** [comment]
Bundle splitting would definitely help but still requires an advanced tooling to be properly done as oppose to CDN that is fairly easy to use without any effort.

**Comment by ljharb** [comment]
If it’s nkt worth the advanced setup, then your traffic is small enough that the CDN isn’t giving you any appreciable benefit. Don’t solve problems before you have them.
